### PR TITLE
Add project skeleton for further development

### PR DESCRIPTION
**Pull request by ikalnitsky**
This commit introduces basic facilities for developing Single Page
Applications (SPA) in ClojureScript. The following things were added:

* Leiningen's `project.clj` for building application as well as helping to
  develop it by running development server. It contains two build
  configurations: one for development, one for production.

* Sample reagent component with a "Hello World" block element.

* Application core that

  * implements routing mechanism
  * renders the "Hello World" reagent component for root "/" route

### CODE REVIEW

**Comment by ikalnitsky** [comment]
I need your review, guys! /cc @olha-kurkaiedova @malor

File: project.clj [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  :description "Web UI for XSnippet code sharing service."
+  :url "https://github.com/xsnippet/xsnippet-web"
+
+  :license
+    {:name "MIT License"
+     :url "https://raw.githubusercontent.com/xsnippet/xsnippet-web/master/LICENSE"
+     :distribution :repo}
+
+  :dependencies
+    [[org.clojure/clojure "1.7.0"]
```
Comment (malor) [inline project.clj]: any particular reason not to use the latest versions here? 1.8.0 and 1.9.14 respectively

Comment (ikalnitsky) [inline project.clj]: Ehm.. Nope, I just copy pasted it from some template. :) I'll fix it in my next patchset.

**Comment by malor** [comment]
Gave it a try, it's a pure magic, but it works :P

**Pull request merged**